sdk/dotnet/Pulumi/X/Automation/PluginInfo.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace Pulumi.X.Automation
+{
+    public class PluginInfo
+    {
+        public string Name { get; }
+
+        public string Path { get; }
+
+        public PluginKind Kind { get; }
+
+        public string? Version { get; }
+
+        public long Size { get; } // TODO: or double? will know once get to implementation
+
+        public DateTime InstallTime { get; } // TODO: is this UTC or local?
```

Comment:
Are these `DateTime` values returned from the CLI as UTC or local?

Reply:
I don't recall off hand if the values are UTC or local (will need to check), but regardless, these properties should be typed as `DateTimeOffset` rather than `DateTime` (and then no need to suffix `Utc`).